src/features/retreat/components/blacksmith/RetreatBlacksmith.tsx

```diff
@@ -44,7 +45,9 @@ export const RetreatBlacksmith: React.FC = () => {
           <Action className="pointer-events-none" text="Craft" icon={icon} />
         </div>
       </div>
-      <ItemsModal isOpen={isOpen} onClose={() => setIsOpen(false)} />
+      <Modal isOpen={isOpen} onClose={() => setIsOpen(false)}>
```

Comment:
```suggestion
      <Modal centered show={isOpen} onHide={() => setIsOpen(false)}>
```

`isOpen` and `onClose` don't exist on Modal component.